directories/src/redux/epics/treasures-epics.js

```diff
@@ -0,0 +1,63 @@
+import { Observable } from "rxjs";
+import { combineEpics } from "redux-observable";
+import moment from "moment";
+import _ from "lodash";
+
+import nodeActions from "../actions/node-actions";
+
+import brokerNode from "../services/broker-node";
+import iota from "../services/iota";
+import ethereum from "../services/ethereum";
+
+const claimTreasureEpic = (action$, store) => {
+  return action$.ofType(nodeActions.NODE_CLAIM_TREASURE).mergeMap(action => {
+    return Observable.fromPromise(brokerNode.treasures())
+      .mergeMap(({ data }) => {
+        const {
+          receiverEthAdd,
+          treasure: { genesisHash, sectorIdx, ethAddr, ethKey }
+        } = data;
+
+        const specialChunkAddress =
+          "HT9MZQXKVBVT9AYVTISCLELYWXTILJDIMHFQRGS9YIJUIRSSNRZFIZCHYHQHKZIPGYYCSUSARFNSXD9UY";
+
+        return Observable.fromPromise(
+          iota.checkIfClaimed(specialChunkAddress)
```

Comment:
This "checkIfClaimed" method is needed for the flow to determine if we should do PoW on a sector or not.  You wouldn't use this for claiming treasure.